src/arrays/programs_involving_data_sequences/programs_involving_multidimensional_data/medium/q2/MultiplyArrays.java

```diff
@@ -0,0 +1,41 @@
+package arrays.programs_involving_data_sequences.programs_involving_multidimensional_data.medium.q2;
+
+/*
+Write a Java program that defines a 2D array of size 2 by 3 with integers in the range of 1 and 5 (inclusive of both
```

Comment:
Change question text to:
`Write a Java program that initializes a 2 x 3 2D array with values {{2, 3, 4}, {4, 5, 1}}. Next, the program needs to randomly generate an integer multiplier in the range of 5 and 10 (inclusive of both values). The program prints the multiplier then calls a method increaseBy that takes the 2D array and the multiplier as input, multiplies all the elements in the 2D array by the multiplier, and returns nothing as output. Lastly, the program displays all the elements in the array after the multiplier is applied. Note that you increaseBy method should be generalized to work with any 2D array length.`